operate/qa/backup-restore-tests/src/test/java/io/camunda/operate/qa/backup/BackupRestoreTest.java

```diff
@@ -145,7 +145,7 @@ private void startAllApps() throws IOException {
     createSnapshotRepository(testContext);
 
     String zeebeVersion = ZeebeClient.class.getPackage().getImplementationVersion();
-    if (zeebeVersion.toLowerCase().contains("snapshot")) {
+    if (zeebeVersion == null || zeebeVersion.toLowerCase().contains("snapshot")) {
```

Comment:
useful when running these tests from the IDE

Reply:
:wrench: Add a comment explaining this